pyproject.toml

```diff
@@ -55,9 +42,25 @@ dev = [
     "pylint", # Used for static linting of files
     "pytest-monitor", # Used to run benchmarks on the test suite
     "memory_profiler", # Used to benchmark memory usage
+    "ray",
     "ray[tune]", # Hyperparameter tuning library
     "optuna", # Optimization search algorithm
 ]
+data-generation=[
+    "alerce",
+    "antares_client",
+]
+runtime-ml = [
```

Comment:
can we change this to something that makes it clear these are for the Samplers/fitting step? runtime-ml may confuse users into thinking this is for classifier training